superset/templates/superset/partials/asset_bundle.html

```diff
@@ -16,8 +16,20 @@
   specific language governing permissions and limitations
   under the License.
 #}
-{% block partial_js %}
-  {% for entry in get_unloaded_chunks(js_manifest(filename), loaded_chunks) %}
+{% macro js_bundle(filename) %}
+  {# HTML comment is needed for webpack-dev-server to replace assets
+     with development version #}
+  <!-- Bundle js {{ filename }} START -->
```

Comment:
The downside is this adds such HTML comments even to the production mode. Can probably create a config flag `DISABLE_ASSET_BUNDLE_WATERMARK` to turn this off.